django_graph_api/graphql/request.py

```diff
@@ -82,7 +84,7 @@ def validate(self):
 
         if self.operation_name:
             if self.operation_name not in operations:
-                self.errors.append('No operation found called `{}`'.format(self.operration_name))
```

Comment:
Ugh, my keyboard has a problem where it sometimes double-types `r`. :-/ Clearly I didn't have thorough enough tests...